<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberTemperature</class>
 <widget class="QWidget" name="NumberTemperature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>70</width>
    <height>85</height>
   </rect>
  </property>
  <widget class="QLabel" name="unitDegreeLearn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>35</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Â°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="indoorTemperatureLearn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>35</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/settings/icons/indoor.svg);
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
